### PR TITLE
Update VolSync description in csv

### PR DESCRIPTION
**Pull request by tesshuflower**
Signed-off-by: Tesshu Flower <tflower@redhat.com>

**Describe what this PR does**
Updates the volsync description in the CSV.

- Updated the csv in config/manifest/bases and then ran `make bundle`.

- Will need to cherry-pick into release-0.5

### CODE REVIEW

File: config/manifests/bases/volsync.clusterserviceversion.yaml [hunk 1]
```diff
@@ -22,7 +22,32 @@ spec:
       kind: ReplicationSource
       name: replicationsources.volsync.backube
       version: v1alpha1
-  description: Asynchronous volume replication for Kubernetes CSI storage
+  description: "Asynchronous volume replication for Kubernetes CSI storage\n\n VolSync
```
Comment (JohnStrunk) [inline config/manifests/bases/volsync.clusterserviceversion.yaml:25]: Can we use the ": |" yaml functionality to make this a bit easier to read?
[SnapScheduler example](https://github.com/backube/snapscheduler/blob/a01f63d35ef1a9ee6362706f905d9c235d1b3feb/config/manifests/bases/snapscheduler.clusterserviceversion.yaml#L101-L172)

Comment (tesshuflower) [reply config/manifests/bases/volsync.clusterserviceversion.yaml:25]: I actually originally inserted it in that format you suggested, but as soon as you run `make bundle`, it re-formats it.  Not sure why it doesn't only format the one in bundle/manifests.

I just tried a test with a simpler description and it does preserve the ": |" format.  There must be something in there (I'm guessing length?) that causes it to decide to re-format. I think it's the `operator-sdk generate bundle` step that's doing it.

Comment (JohnStrunk) [reply config/manifests/bases/volsync.clusterserviceversion.yaml:25]: oh, that's really strange. I'm pretty sure the flow is the same w/ SnapScheduler and it doesn't cause a problem.

Comment (tesshuflower) [reply config/manifests/bases/volsync.clusterserviceversion.yaml]: Update: it turned out to be trailing spaces in the bullet points - meant to indicate a line break.  During the yaml conversion this caused it to get reformatted - I've tried again but had to add newlines in-between the bullet points subject and text.

**Pull request merged**